src/lib/isMobilePhone.js

```diff
@@ -20,7 +20,7 @@ const phones = {
   'el-GR': /^(\+?30|0)?(69\d{8})$/,
   'en-AU': /^(\+?61|0)4\d{8}$/,
   'en-GB': /^(\+?44|0)7\d{9}$/,
-  'en-GH': /^(\+233|0)(20|50|24|54|27|57|26|56|23|28)\d{7}$/,
+  'en-GH': /^(((\+233|0)(20|50|24|54|55|27|57|26|56|23|28)\d{7})|((\+233|0)(302|303)\d{6})|((\+233|0)(3035|3120|3121|3122|3123|3124|3125|3126|3220|3221|3222|3223|3224|3225|3320|3321|3322|3323|3420|3421|3431|3423|3424|3425|3426|3427|3428|3429|3430|3520|3521|3522|3523|3524|3525|3526|3527|3620|3621|3623|3624|3625|3626|3720|3721|3722|3723|3724|3725|3726|3820|3821|3822)\d{5}))$/,
```

Comment:
I think we can simplify your regex. Could you list here the phone numbers that were failing to pass before this change?

Reply:
The phone numbers are not consistent. As I said its all listed at https://en.wikipedia.org/wiki/Telephone_numbers_in_Ghana. See my comments above.

Reply:
Oh, I see, you are mixing up _fixed-line_ with mobile, can you check the section listed as _Mobile numbering plan_; which looks like the current regex is ok. Could you give a few `mobile-phone` numbers that are failing the check?

Reply:
The fixed-line numbers are those prefixed with 03X and none of them is currently supported. I am the same person who submitted the current ones and I'm extending it with support for others that were left out. 

Those under the  *Mobile numbering* plan are not the only supported phone numbers. You can even see it quoted like...

>Due to the unreliability of fixed-line infrastructure in Ghana, mobile phones are more widely used. Competition among the various mobile carriers has spurred growth with a subscriber penetration rate of 98% in 2010.[2] The poor call quality of mobile phones, however, means that more people hold more than one mobile phone, usually with two or more different carriers.

The hold is delaying support for fixed-line numbers. Please don't keep us waiting too much...its been almost 2 months since I last submitted this pull request.

Reply:
@aberba i think that you are missing the point that ```isMobilePhone``` is only for mobile numbers validation. Fixed lines are not included/validated in the library